superset/utils/core.py

```diff
@@ -1553,3 +1553,7 @@ class AdhocMetricExpressionType(str, Enum):
 class RowLevelSecurityFilterType(str, Enum):
     REGULAR = "Regular"
     BASE = "Base"
+
+
+def is_test() -> bool:
+    return "SUPERSET_TESTENV" in os.environ
```

Comment:
Here it might be worth considering using `strtobool`, which will parse the string value for known string representations of boolean values so the env variable can also explicitly be set to `True`:

```python
from distutils.util import strtobool 

strtobool(os.environ.get("SUPERSET_TESTENV", "false"))
```

Reply:
Thanks, @villebro! I'll update the PR.